silhouette/Graphtec.py

```diff
@@ -1398,7 +1398,8 @@ def plot(self, mediawidth=210.0, mediaheight=297.0, margintop=None,
         self.draw_mm_cmd(bbox['ury'], bbox['llx'])]
 
     # potentially long command string needs extra care
-    self.safe_send_command(cmd_list)
```

Comment:
Yes I also stumbled over that one, but I think it only occurs if sending nothing to the device, no? If that were the case, this should probably be handled from the plugin itself, i.e. we should test directly when the extensions is launched, whether a object/path is currently selected.

But it probably doesn't hurt to catch it here also. Just saying.